src/services/requests/ito/index.ts

```diff
@@ -0,0 +1,80 @@
+import api from '@/services/api';
+import { IAsset, IITO, IITOsResponse } from '@/types';
+import { getPrecision } from '@/utils/precisionFunctions';
+import { NextRouter } from 'next/router';
+
+export const requestITOs = async (
+  router: NextRouter,
+  pageParam = 1,
+): Promise<any | void> => {
+  const asset = router?.query?.asset ?? '';
+  const isActive = router?.query?.active ?? true;
+  try {
+    const itosResponse = await api.get({
+      route: `ito/list`,
+      query: { page: pageParam, active: isActive, asset },
+    });
+    if (!itosResponse.error || itosResponse.error === '') {
+      return itosResponse;
+    }
+  } catch (error) {
+    console.error(error);
+  }
+  return;
+};
+
+export const requestAssetsList = async (
+  data: IITOsResponse,
+): Promise<any | void> => {
+  if (data) {
```

Comment:
consider checking the length of the data, since it returns an object, will always be true